cfme/provisioning.py

```diff
@@ -35,6 +40,137 @@ def select_security_group(sg):
     sel.sleep(1)
 
 
+class ProvisioningForm(BaseLoggedInPage):
+    @View.nested
+    class request(Tab):  # noqa
+        TAB_NAME = 'Request'
+        email = Input(name='requester__owner_email')
+        first_name = Input(name='requester__owner_first_name')
+        last_name = Input(name='requester__owner_last_name')
+        notes = Input(name='requester__request_notes')
+        manager_name = Input(name='requester__owner_manager')
+
+    @View.nested
+    class purpose(Tab):  # noqa
+        TAB_NAME = 'Purpose'
+        apply_tags = VersionPick({
+            Version.lowest(): CheckboxSelect('//div[@id="all_tags_treebox"]//ul'),
+            '5.7': BootstrapTreeview('all_tags_treebox')})
+
+    @View.nested
+    class catalog(Tab):  # noqa
+        TAB_NAME = 'Catalog'
+        num_instances = BootstrapSelect('service__number_of_vms')
+        instance_name = Input(name='service__vm_name')
+        instance_description = Input(name='service__vm_description')
+        vm_filter = BootstrapSelect('service__vm_filter')
+        num_vms = BootstrapSelect('service__number_of_vms')
+        vm_name = Input(name='service__vm_name')
```

Comment:
Agreed, I didn't audit the form too much from its existing fields. I'll condense these though.